packages/snap-controller/src/Search/SearchController.ts

```diff
@@ -89,6 +89,7 @@ export class SearchController extends AbstractController {
 			}
 		>;
 	} = { product: {} };
+	private schemaMap: Record<string, SearchSchemaData> = {};
```

Comment:
If possible it would be good not to expose this as a property on the class (private in TS is not enough to prevent people from using it or modifying its contents).